src/lib/import_export/csv_meta.hpp

```diff
@@ -22,12 +22,16 @@ struct ParseConfig {
   char escape = '"';
   char delimiter_escape = '\\';
 
-  // If this is set to true, "4.3" will not be accepted as a value for a float column
+  // If this is set to true, "4.3" will not be accepted as a value for a float column.
   bool reject_quoted_nonstrings = true;
 
-  // If this is set to true, an unquoted null string causes an exception (only empty field is allowed as null value)
+  // If this is set to true, an unquoted null string causes an exception (only empty field is allowed as null value).
   bool reject_null_strings = true;
 
+  // If this is set to true, "...,Null,..." would be parsed as "Null"-string, otherwise it's a null value.
+  // Has no effect if reject_null_strings is true.
+  bool unquoted_null_string_is_string = false;
```

Comment:
I tried to do that, but gave up after a while; parsing an enum from json is not as easy as it might seem and I decided it is not worth the effort. I can do it though if you think it is important enough.

Reply:
Mmh, I reviewed this in two sessions. Duplicate of the first comment.